backend/app/db/schemas/search.py

```diff
@@ -35,7 +34,7 @@ class SearchRequestBody(BaseModel):
     year_range: Optional[Tuple[Optional[int], Optional[int]]] = None
 
     sort_field: SortField = SortField.DATE
```

Comment:
Does this mean that the default sort order of search results is date descending? It should be score descending, right?

Reply:
I really should keep a more careful eye on autocomplete